pubspec.yaml

```diff
@@ -25,8 +25,14 @@ flutter:
   # be modified. They are used by the tooling to maintain consistency when
   # adding or updating assets for this project.
   plugin:
-    androidPackage: com.linusu.flutter_web_auth
-    pluginClass: FlutterWebAuthPlugin
+    platforms:
```

Comment:
This breaks older versions of Flutter so I will release this as a breaking change